docs/sphinx/dev_guide/host_config.rst

```diff
@@ -1,12 +1,4 @@
-.. ##
-.. ## Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
-.. ## and RAJA project contributors. See the RAJA/COPYRIGHT file
-.. ## for details.
-.. ##
-.. ## SPDX-License-Identifier: (BSD-3-Clause)
-.. ##
-
-.. raja_host_config:
+.. host_config:
```

Comment:
Please don't remove the copyright statement from files.

Also, please add it to the other files you added.